packages/kit/test/apps/basics/test/test.js

```diff
@@ -1224,7 +1224,7 @@ test.describe.parallel('Headers', () => {
 	});
 });
 
-test.describe.parallel('Load', () => {
+test.describe('Load', () => {
```

Comment:
What if we only make this one serial and leave the rest parallel? Does that fix the issue? It seems like there's at least some amount of strangeness coming from this second server, so it might be worth a shot

Reply:
Nevermind. That's not it. If I comment out the tests that start new servers I still get hangs